ha/setup/cluster_validator/__init__.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
```

Comment:
I think python3 not need __init__.py
it create __pycache__

Reply:
I checked that, thanks. Python 3 still needs `__init__.py` for normal packages. However it's not this case. The package will be implemented in separate feature, so i deleted this file from PR.